.pre-commit-config.yaml

```diff
@@ -26,5 +26,19 @@ repos:
                 name: flake8-cython
                 args: ["--config=python/cuspatial/.flake8.cython"]
                 types: [cython]
+      - repo: local
+        hooks:
+              - id: clang-format
+                # Using the pre-commit stage to simplify invocation of all
+                # other hooks simultaneously (via any other hook stage).  This
+                # can be removed if we also move to running clang-format
+                # entirely through pre-commit.
+                stages: [commit]
```

Comment:
FYI this isn't necessary, the stages are only used in cudf because we actually run our linting in CI via pre-commit but we want to avoid using clang-format. There's no harm in leaving it though and maybe we'll change cuspatial CI eventually too.

Reply:
TIL. Thanks!